### PR TITLE
docs(changelog): for release v14.0.0 [skip ci]

### PR DESCRIPTION
**Pull request by emmenko**
In preparation for the release. Let me know if it's not clear enough or if I forgot something.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,215 @@
+## [14.0.0](https://github.com/commercetools/merchant-center-application-kit/compare/v13.13.3...v14.0.0) (2019-07-18)
+
+This release introduces several **breaking changes**, as documented below, mostly around React Hooks and permissions.
```
Comment (tdeekens) [inline CHANGELOG.md]:
```suggestion
This release introduces several **breaking changes**, as documented below, mostly around React Hooks and our own permission functionality.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,21 +1,215 @@
...
+
+> You can read the v14 announcement and version tracker here: https://github.com/commercetools/merchant-center-application-kit/issues/748
+
+### Full support for React Hooks
+
+We bumped some peer dependencies that provide full hooks support, so you need to update those:
```
Comment (tdeekens) [inline CHANGELOG.md]:
```suggestion
In order to support hooks the following peer dependencies have to be updated:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,21 +1,215 @@
...
+
+- `react>=16.8`
+- `react-dom>=16.8`
+- `react-intl>=3` (_currently still in beta_)
+- `react-redux>=7`
+
+_If you're not familiar with React Hooks, we recommend to have a look at the following resources, as Hooks are now the recommended way to write components:_
```
Comment (tdeekens) [inline CHANGELOG.md]:
```suggestion
_If need more information about React Hooks, we recommend to have a look at the following resources. Hooks are another way to write components and come with many benefits:_
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,21 +1,215 @@
...
+
+- https://reactjs.org/docs/hooks-intro.html
+- https://egghead.io/courses/simplify-react-apps-with-react-hooks
+- https://usehooks.com/
+
+_TL;DR; using Hooks makes the code more readable, easier to test, less code to ship_
```
Comment (tdeekens) [inline CHANGELOG.md]:
```suggestion
_TL;DR; with Hooks code can become more readable, easier to test while often improving performance and reducing bundlesize.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,21 +1,215 @@
...
+  - `ServicePageResponseLayout` (was _deprecated_)
+- `permissions`:
+  - `hasPermission`
+  - `permissions`
+- `react-notifications`:
+  - `GetCustomNotificationComponent`
+
+### Dropped support for IE11
+
+We don't support IE 11 anymore, therefore we removed polyfills and things related to IE 11 support.
```
Comment (tdeekens) [inline CHANGELOG.md]:
```suggestion
Due to IE11's low usage and a high penality on mondern browsers due to transpilation and polyfills, we decided not to support IE 11 anymore. As a result polyfills and other things related such as transpilation were removed or adjusted.
```

**Comment by emmenko** [comment]
I will manually release a canary/beta version today, so that I can start migrating the apps and check that things are working.

**Pull request merged**